writer_test.go

```diff
@@ -1225,6 +1225,27 @@ func ExampleMediaPlaylist_Segments_scte35_oatcls() {
 	// media2.ts
 }
 
+func ExampleMediaPlaylist_Segments_scte35_cue() {
+	f, _ := os.Open("sample-playlists/media-playlist-with-cue-scte35.m3u8")
+	p, _, _ := DecodeFrom(bufio.NewReader(f), true)
+	pp := p.(*MediaPlaylist)
+	fmt.Print(pp)
```

Comment:
A test should not use `fmt.Print`. Please use https://pkg.go.dev/testing#T.Log instead (or delete the log line altogether, since it doesn't seem particularly useful)

Reply:
Removed this test and created a new one that tests the same functionality in: https://github.com/zencoder/m3u8/commit/c953678b1aa9fccf3c8ab6bb7b540872f75ba4c0